class-gf-razorpay.php

```diff
@@ -0,0 +1,255 @@
+        return array(
+            array(
+                'title'       => '',
+                'fields'      => array(
+                    array(
+                        'name'    => 'gf_razorpay_key',
+                        'label'   => esc_html__( 'Razorpay Key', 'razorpay-gravity-forms' ),
+                        'type'    => 'text',
+                        'class'   => 'medium',
+                    ),
+                    array(
+                        'name'    => 'gf_razorpay_secret',
+                        'label'   => esc_html__( 'Razorpay Secret', 'razorpay-gravity-forms' ),
+                        'type'    => 'text',
+                        'class'   => 'medium',
+                    ),
+                    array(
+                        'type' => 'save',
+                        'messages' => array(
+                            'success' => esc_html__( 'Settings have been updated.', 'razorpay-gravity-forms' )
+                        ),
+                    ),
+                ),
+            ),
+        );
+    }
+
+    public function get_customer_fields_array($feed, $entry)
+    {
+        $fields = array();
+
+        foreach ($this->get_customer_fields() as $field)
+        {
+            $field_id = $feed['meta'][$field['meta_name']];
+
+            $value = rgar($entry, $field_id);
+
+            $fields[$field['meta_name']] = $value;
+        }
+
+        return $fields;
+    }
+
+    public function get_customer_fields() {
+        return array(
+            array( 'name' => 'first_name', 'label' => 'First Name', 'meta_name' => 'billingInformation_firstName' ),
+            array( 'name' => 'last_name', 'label' => 'Last Name', 'meta_name' => 'billingInformation_lastName' ),
+            array( 'name' => 'email', 'label' => 'Email', 'meta_name' => 'billingInformation_email' ),
+            array( 'name' => 'address1', 'label' => 'Address', 'meta_name' => 'billingInformation_address' ),
+            array( 'name' => 'address2', 'label' => 'Address 2', 'meta_name' => 'billingInformation_address2' ),
+            array( 'name' => 'city', 'label' => 'City', 'meta_name' => 'billingInformation_city' ),
+            array( 'name' => 'state', 'label' => 'State', 'meta_name' => 'billingInformation_state' ),
+            array( 'name' => 'zip', 'label' => 'Zip', 'meta_name' => 'billingInformation_zip' ),
+            array( 'name' => 'country', 'label' => 'Country', 'meta_name' => 'billingInformation_country' ),
+        );
+    }
+
+    public function callback() {
+        $entry_id = $_COOKIE['entry_id'];
+
+        $razorpay_order_id = $_COOKIE['razorpay_order_id'];
+
+        $entry = GFAPI::get_entry($entry_id);
+
+        $api = new Api($this->get_plugin_setting('gf_razorpay_key'),
+            $this->get_plugin_setting('gf_razorpay_secret'));
+
+        $attributes = array (
+            'razorpay_order_id'   => $razorpay_order_id,
+            'razorpay_payment_id' => rgpost('razorpay_payment_id'),
```

Comment:
@riagarg5 You're right. But that wasn't the point I was trying to make. You see, for any post / get parameter, wordpress plugins require you to validate / clean the data before proceeding further. The link I gave to you does that, and this plugin needs to do the same. cc @captn3m0